apps/web/pages/my-account/new-return/[id].vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <UiDivider class="col-span-3 -mx-4 !w-auto md:mx-0" />
+  <h2
+    v-if="!confirmation"
+    class="hidden md:block col-span-3 typography-headline-4 font-bold mx-4 capitalize"
+    data-testid="account-new-return-heading"
+  >
+    {{ t('returns.return') }}
+  </h2>
+  <div v-if="!confirmation" class="col-span-3 mt-8 !w-auto">
```

Comment:
<img width="1022" alt="Bildschirmfoto 2024-02-29 um 16 24 49" src="https://github.com/plentymarkets/plentyshop-pwa/assets/30629157/8eb7950f-4c9a-4a3b-a8fa-8a9ae9bf1a08">

Should have the same margin as the heading